itunesCall.js

```diff
@@ -0,0 +1,23 @@
+const fetch = require("node-fetch");
+
+const getPodcastData = async () => {
+    try {
+		const response = await fetch(`https://itunes.apple.com/search?term=$podcast`, {
+			"method": "GET",
```

Comment:
We all discussed briefly if GET method was inherent on fetch requests, but maybe node-fetch makes that not the case? We can definitely do some research on this to see if this code can be refined or not

Reply:
yeah, this seems like a new concept.  It's possible that this is directly related to backend...maybe it has always happened behind the scenes from the frontend view.

Reply:
When we write the table for Movies we can try without  the `GET` and see what happens.